src/index.js

```diff
@@ -169,7 +172,7 @@ export default function renderToString(vnode, context, opts, inner) {
 		for (let i=0; i<len; i++) {
 			let child = children[i];
 			if (!falsey(child)) {
-				let ret = renderToString(child, context, opts, true);
+				let ret = renderToString(child, context, opts, true, nodeName==='svg');
```

Comment:
only thing I just noticed - I think we need to toggle into and out of SVG mode the same [way Preact's renderer does](https://github.com/developit/preact/blob/master/src/vdom/diff.js#L112):

```js
isSvgMode = nodeName==='svg' ? true : nodeName==='foreignObject' ? false : isSvgMode;
```

The reason for this is so that the String render switches into SVG mode when it enters into an `<svg>` element, but then switches back out of SVG mode if it leaves SVG via a `<foreignObject>` element.  Let me know if that makes sense :)